app/assets/stylesheets/blacklight.scss

```diff
@@ -1,5 +1,10 @@
 @import 'bootstrap';
 
 @import 'blacklight/blacklight';
-/* @import 'blacklight_marc'; */
+$padding-xs-horizontal: 1ch !default;
 
+@import 'blacklight_marc';
+
+#marc_view .tag {
```

Comment:
This is only temporary; bootstrap 4 alpha 6 will apparently rename their new `.tag` styles back to `.badge`